packages/react/src/runtime/style.tsx

```diff
@@ -47,23 +48,19 @@ function Style(props: StyleProps) {
       }
 
       return (
-        <style nonce={props.nonce}>
+        <style data-cmpld nonce={props.nonce}>
```

Comment:
Any short alternative to `data-cmpld`? `data-c`?

Reply:
i wanted something that wouldnt be used by somewhere else - we could go down to three? `data-cmp`

Reply:
Not sure..lets stick to `data-cmpld` as of now.